.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -26,6 +28,7 @@ body:
       If the code is too long (hopefully, it isn't), feel free to put it in a public gist and link it in the issue: https://gist.github.com.
 
       Please also paste or describe the results you observe instead of the expected results. If you observe an error, please paste the error message including the **full** traceback of the exception. It may be relevant to wrap error messages in ```` ```triple quotes blocks``` ````.
+      If your issue is related to numerical accuracy or reproducibility, please read the [numerical accuracy](https://docs.pytorch.org/docs/stable/notes/numerical_accuracy.html) and [reproducibility](https://docs.pytorch.org/docs/stable/notes/randomness.html) notes.
```

Comment:
Do you mind adding a quick followup that reporter should provide some justification why one output is correct and other is not, or what correct output should look like (with reference to some paper if possible)